addons/xbmc.python/addon.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<addon id="xbmc.python" version="2.24.0" provider-name="Team-Kodi">
-  <backwards-compatibility abi="2.1.0"/>
+<addon id="xbmc.python" version="2.25.0" provider-name="Team-Kodi">
+  <backwards-compatibility abi="2.25.0"/>
```

Comment:
just out of interest, couldn't we add a compatibility layer/wrapper or simmulate the old behavior? I have to agree that a deprecation period for add-on devs would be a good thing to have. And I know it's python, but IIRC @AlwinEsch has been working on a new binary add-on interface that also support compatibility layers, could we make use of that layer for the python ABI as well by moving it to a binary add-on? I know that this likely is a huge task and ofc outside of the scope of this PR, but maybe worth the effort on the long run?